NAMESPACE

```diff
@@ -23,11 +23,16 @@ export(show.project)
 export(stub.tests)
 export(test.project)
 export(translate.dcf)
+export(type_sum.data_reader)
+importFrom(R6,R6Class)
 importFrom(stats,setNames)
+importFrom(tibble,data_frame)
```

Comment:
Just checking — all instances of `data.frame` should now be `data_frame`?

Reply:
I changed them in `list.data`, and since we already convert everything that's not a `data.table` almost all cases should be converted now. We can of course check the rest of the code base for instances that should be converted.